packages/web-player-component/src/index.js

```diff
@@ -0,0 +1,27 @@
+import webplayer from '@eyevinn/web-player';
+import style from '@eyevinn/web-player/dist/webplayer.css';
+
+export default class PlayerComponent extends HTMLElement {
+  constructor() {
+    //Call constructor of HTMLElement
+    super();
+    //Create shadow root
+    this.attachShadow({ mode: 'open' });
+    const { shadowRoot } = this;
```

Comment:
We should discuss whether we should use a shadow dom at all vs render in the main document.